<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageUser</class>
 <widget class="QDialog" name="ManageUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quản lí tài khoản</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupAction">
     <property name="title">
      <string>Thao tác</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupButton">
        <property name="title">
         <string>Thao tác</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>Thêm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdate">
           <property name="text">
            <string>Cập nhật</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>Xóa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnReset">
           <property name="text">
            <string>Nhập lại</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupInput">
        <property name="title">
         <string>Thông tin</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="7" column="0">
          <widget class="QLabel" name="lbRole">
           <property name="text">
            <string>Quyền</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLineEdit" name="inputUsername"/>
         </item>
         <item row="7" column="2">
          <widget class="QComboBox" name="comboRole"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="lbUserName">
           <property name="text">
            <string>Tài khoản</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbId">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbFullname">
           <property name="text">
            <string>Họ và tên</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="inputFullname"/>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="inputID">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbGender">
           <property name="text">
            <string>Giới tính</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbEmail">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="comboGender">
           <item>
            <property name="text">
             <string>Nam</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nữ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Không xác định</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lbBirthday">
           <property name="text">
            <string>Ngày sinh</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QDateEdit" name="inputBirthday"/>
         </item>
         <item row="9" column="2">
          <widget class="QLineEdit" name="inputPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="lbPassword">
           <property name="text">
            <string>Mật khẩu</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbPhone">
           <property name="text">
            <string>Điện thoại</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="inputPhone"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="inputEmail"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lbAddress">
           <property name="text">
            <string>Địa chỉ</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QTextEdit" name="inputAddress"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupSearch">
        <property name="title">
         <string>Tìm kiếm</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupSearchBy">
           <property name="title">
            <string>Tìm kiếm theo</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioFullanme">
              <property name="text">
               <string>Tên</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioID">
              <property name="text">
               <string>ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioPhone">
              <property name="text">
               <string>Số điện thoại</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupKeyWord">
           <property name="title">
            <string>Từ khóa</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="inputSearch">
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSearch">
              <property name="text">
               <string>Tìm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTable">
     <property name="title">
      <string>Tìm kiếm</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Sắp xếp</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1">
         <item>
          <widget class="QComboBox" name="comboSort">
           <item>
            <property name="text">
             <string>Không sắp xếp</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sắp xếp theo tên</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sắp xếp theo quyền</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sắp xếp theo ngày tạo</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSort">
           <property name="text">
            <string>Sắp xếp</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableUser"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
